fecgov/fec-cms
# Feature/3305 social images and meta update

## Summary

- Resolves #3305 
- Updates the filename of the image for the data pages
- Adds the page path to the og:url tag for all pages that include meta-tags.jinja (/data/ etc)
- Images have been re-encoded and exported by Photoshop rather than downloading and saving from Github

## Impacted areas of the application
Will only affect the meta information across the site.

## Screenshots
None

## Related PRs
None

## How to test
- pull, `npm i`, `npm run build`, `./manage.py runserver`
- All of the updates will be inside the meta tags in the source code of the page
- Check that the `og:image` value ends with `/fec-data.png` instead of `/data.png` for
  - [homepage](http://127.0.0.1:8000/)
  - [data](http://127.0.0.1:8000/data/)
  - [raising-bythenumbers](http://127.0.0.1:8000/data/raising-bythenumbers/)
  - other data pages
- The `og:url` value should be the location of the page on production (not just `https://www.fec.gov` but `https://www.fec.gov/data/`, `https://www.fec.gov/data/raising-bythenumbers/`, etc.) for
  - [candidates](http://127.0.0.1:8000/data/browse-data/?tab=candidates)
  - [committees](http://127.0.0.1:8000/data/browse-data/?tab=committees)
  - [elections](http://127.0.0.1:8000/data/elections/?state=&cycle=2020&election_full=true)
  - [search results](http://127.0.0.1:8000/search/?type=candidates&type=committees&type=site&query=test)
  - [datatables](http://127.0.0.1:8000/data/receipts/individual-contributions/?two_year_transaction_period=2020&min_date=01%2F01%2F2019&max_date=12%2F31%2F2020)
  - etc
- If you're the last approval, remove "Please review", merge, delete

____
